### PR TITLE
New packages: biblioteq-2018.05.05 and yaz-5.25.0 (closes #13554)

### LINKED ISSUE #13554: package request:biblioteq
https://github.com/textbrowser/biblioteq

since tellico dint get much love maybe this will get some

### PR DESCRIPTION
**Pull request by cr6git**

### CODE REVIEW

File: srcpkgs/biblioteq/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+version=2017.08.19
+revision=1
+build_style=qmake
+configure_args="-o Makefile biblioteq.pro"
+conf_files="/etc/biblioteq.conf"
+makedepends="libpqxx-devel qt5-devel poppler-cpp-devel poppler-qt5-devel
+ sqlite-devel yaz-devel"
+short_desc="Professional cataloging and library management suite"
+maintainer="cr6git <quark6@protonmail.com>"
+license="BSD"
```
Comment (maxice8) [inline srcpkgs/biblioteq/template]: please use SPDX identifier, BSD is ***extremely*** ambiguous.

File: srcpkgs/yaz/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pkgname=yaz
+version=5.23.1
+revision=1
+build_style=gnu-configure
+configure_args="--enable-shared=yaz"
+hostmakedepends="bison pkg-config"
+makedepends="gnutls-devel libxslt-devel"
+short_desc="Toolkit supporting development of Z39.50/SRW/SRU clients and servers"
+maintainer="cr6git <quark6@protonmail.com>"
+license="BSD"
```
Comment (maxice8) [inline srcpkgs/yaz/template]: ditto

**Comment by cr6git** [comment]
The build failed on these two archs:
````
armv7l-linux-gnueabihf-g++: error: unrecognized argument in option '-mtune=generic'
arm-linux-musleabihf-g++: error: unrecognized argument in option '-mtune=generic'
````
:confused:

**Comment by newbluemoon** [comment]
You can remove `-mtune=generic` in `biblioteq.pro`. Perhaps remove it for all targets but i686* and x86_64*? Tested it for armv7l, there you’ll also have to remove `-Werror`:

`sed -i -e 's/-mtune=generic//' -e 's/-Werror//' biblioteq.pro`

**Comment by abenson** [comment]
I didn't see this PR and started working on my own -- FYI new releases have dropped for both (biblioteq-2018.05.05 and yaz-5.25.0).

**Comment by cr6git** [comment]
Thanks @abenson , I will update my templates.

**Comment by cr6git** [comment]
it's done.

**Comment by cr6git** [comment]
Replaced by https://github.com/void-linux/void-packages/pull/899

**Pull request closed without merging**